configuration/multimedia

```diff
@@ -1,136 +1,160 @@
-   In Debian Template VM:
-   sudo apt-get install qubes-usb-proxy
+`cp -p /usr/share/spotify/spotify.desktop /usr/share/applications/`
+`cp /usr/share/spotify/icons/spotify-linux-16.png /usr/share/icons/hicolor/16x16/apps/spotify.png`
 
-2) Install libdbdread4 from the default repositories:
-   In Debian Template VM:
-   apt-get install libdvdread4
-   
-3) Install libdvdcss for the decryption of CSS protected-DVDs.
-   Because of license restrictions in various countries this can't be done from the Debian repositories but needs to be installed manually.
-   (not that hard :-)).
-   Download the packaged .deb from: http://www.deb-multimedia.org/dists/stable/main/binary-amd64/package/libdvdcss2
-   In Debian Template VM:
-   wget http://www.deb-multimedia.org/pool/main/libd/libdvdcss/libdvdcss2_1.3.0-dmo1_amd64.deb
-   sudo dpkg -i libdvdcss2_1.3.0-dmo1_amd64.deb
 
-4) I like to use VLC to playback Video/DVDs - you might want to use another player, but VLC works fine.
-   As this is not installed in the default Debian Template:
-   In Debian Template VM:
-   sudo apt-get install vlc
-   This will also install some dependencies.
+Installation of VLC
+-------------------
 
-5) Shutdown the Debian Template VM and launch your multimedia App VM
+To play DVDs you can install VLC with the needed Codecs
 
-6) Plugin your external USB DVD-Drive (if you don't have an internal drive) and pass it to the multimedia App VM
-   In dom0:
-   qvm-usb -a multimedia sys-usb:3-6
+Add Repository for libdvdcss
+(See also: http://www.videolan.org/developers/libdvdcss.html)
 
-7) Launch VLC
-   qvm-run multimedia vlc
-   Warning: a window will open in the background which will ask for collecting metadata information.
-   I haven't seen the window as it was covered by the VLC main window, so that I couldn't launch any commands in this window.
+Add GPG-Key
+`apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 8F0845FE77B16294429A79346BCA5E4DB84288D9`
 
-8) Open DVD within VLC:
-   Media > Open Disk
-   You can leave the defaults (/dev/sr0) and just hit play
+> Public-Key: http://keyserver.ubuntu.com/pks/lookup?op=vindex&search=0xB84288D9&fingerprint=on
+> Fingerprint: 8F08 45FE 77B1 6294 429A  7934 6BCA 5E4D B842 88D9 
 
-Enjoy your DVD
+Add repositories to install VLC and libdvdcss
+`echo "deb http://download.videolan.org/pub/debian/stable/ /" >> /etc/apt/sources.list`
+`echo "deb-src http://download.videolan.org/pub/debian/stable/ /" >> /etc/apt/sources.list`
+
+Update package repositories
+`apt-get update`
+
+Install libdvdcss and VLC
+`apt-get install -y libdvdcss2 vlc` 
+
+
+ 
+Installation Google Chrome
+--------------------------
+
+To play Videos with Netflix, Amazon Prime & Co using Chrome is a good option as it has all needed codecs included.
+Hint: Using Chromium will not work for some reasons.
```

Comment:
Is it because Google Chrome has (had?) flash built-in, whereas Chromium doesn't?

Reply:
I was trying out Chromium before using Chrome, but it seems that Chromium is "missing" some protected DRM. 
See also here: https://ubuntuforums.org/showthread.php?t=2327128